src/main/java/com/google/security/zynamics/reil/algorithms/mono2/common/MonoReilSolver.java

```diff
@@ -63,17 +63,17 @@
    * result we skip edges which have not been visited in order to omit artificially inserted entry
    * and exit edges.
    */
-  private final Set<IInstructionGraphEdge> m_traversedEdges = new HashSet<IInstructionGraphEdge>();
+  private final Set<IInstructionGraphEdge> m_traversedEdges = new HashSet<>();
 
   public MonoReilSolver(final IInstructionGraph instructionGraph,
       final AnalysisDirection analysisDirection, final ILattice<LatticeElementType> lattice) {
     m_graph = Preconditions.checkNotNull(instructionGraph,
-        "Error: instruction graph argument can not be null");
```

Comment:
Ok I will need to make him aware of this fact then. To give you some background there is a script which assigns unique error number to these messages prefixed by error if they have not yet gotten a number assigned. Therefore traces are much more precise then just the error message which might be duplicated.